clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanSubsystemXMLReader.java

```diff
@@ -164,6 +170,30 @@ private void parseContainer(XMLExtendedStreamReader reader, PathAddress subsyste
                     this.parseTransport(reader, address, operations);
                     break;
                 }
+                case EVICTION_THREAD_POOL: {
+                    if (this.schema.since(InfinispanSchema.VERSION_3_0)) {
```

Comment:
Shouldn't these all throw exceptions if the schema is wrong? At the moment it will just fall down into the next case statement.